src/NuGet.Status/NuGet.Status.csproj

```diff
@@ -171,7 +171,7 @@
       <Version>1.1.3</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Owin.Host.SystemWeb">
-      <Version>3.0.1</Version>
+      <Version>3.1.0</Version>
```

Comment:
Should we pull up the rest of the owin dependencies here?

Reply:
I tried upgrading all OWIN dependencies to 4.1.0 which is latest, but it didn't work for some reason. Not sure if it breaks with more recent versions. From what I gather online with other folks who have seen this issue, addressing the fix above seems to have helped. I would rather have this minimal fix right now and if it doesn't address the issue then consider other options, given its difficult to reproduce.